src/todo-list-core/Security/TodoApiClaimTypes.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace TodoListAPI.Security
+{
```

Comment:
extending claim types with custom values and using namespace to avoid potential collisions with other claim providers